classes/Cron.php

```diff
@@ -236,9 +257,9 @@ public static function parseWebmention($request)
             // create the commention data
             $finaldata = [
                 'status' => Commentions::defaultstatus($result['type']),
-                'name' => $result['author']['name'],
-                'website' => $result['author']['url'],
-                'avatar' => $result['author']['photo'],
+                'name' => $result['author']['name'] ?? null,
+                'website' => $result['author']['url'] ?? null,
+                'avatar' => $result['author']['photo'] ?? null,
```

Comment:
The `php-comments`lib returns `false` for each of these values by default (see https://github.com/indieweb/php-comments/blob/master/src/indieweb/comments.php#L66), if they’re empty. As far as I understand, if these values are missing from the webmention source, the returned value will be `false`. But if the `url` field does not validate and thus is unset in line 164, its value in the `$finaldata` array would be `null`. Am I right with this?

Reply:
Yes, this is inconsistent indeed. The outcome should be the same as `Commentions::sanitize()` checks for `empty()` (so false or null lead to the same result), but I cleaned it up and updated the PR.